plugin-nabu/src/main/java/ca/on/oicr/gsi/shesmu/nabu/NabuCaseArchiveValue.java

```diff
@@ -0,0 +1,101 @@
+package ca.on.oicr.gsi.shesmu.nabu;
+
+import ca.on.oicr.gsi.shesmu.plugin.input.ShesmuVariable;
+import java.time.Instant;
+import java.util.Optional;
+import java.util.Set;
+
+public class NabuCaseArchiveValue {
+  private final Optional<Instant> caseFilesUnloaded;
+  private final String caseIdentifier;
+  private final Optional<String> commvaultBackupJobId;
+  private final Instant created;
+  private final Optional<Instant> filesCopiedToOffsiteArchiveStagingDir;
+  private final Optional<Instant> filesLoadedIntoVidarrArchival;
+  private final Set<String> limsIds;
+  private final Instant modified;
+  private final long requisitionId;
+  private final Set<String> workflowRunIdsForOffsiteArchive;
+  private final Optional<Set<String>> workflowRunIdsForVidarrArchival;
+
+  public NabuCaseArchiveValue(
+      Optional<Instant> caseFilesUnloaded,
+      String caseIdentifier,
+      Optional<String> commvaultBackupJobId,
+      Instant created,
+      Optional<Instant> filesCopiedToOffsiteArchiveStagingDir,
+      Optional<Instant> filesLoadedIntoVidarrArchival,
+      Set<String> limsIds,
+      Instant modified,
+      long requisitionId,
+      Set<String> workflowRunIdsForOffsiteArchive,
+      Optional<Set<String>> workflowRunIdsForVidarrArchival) {
+    super();
```

Comment:
What's the role of super() here? in my IDE it opens Object.class

Reply:
I'm not exactly sure; this is following the same pattern used by other `*Value` classes (e.g. [PineryIUSValue](https://github.com/oicr-gsi/shesmu/blob/d2412f304ffb88b5884aa5c500fdbe4e2b003ea0/plugin-pinery/src/main/java/ca/on/oicr/gsi/shesmu/pinery/PineryIUSValue.java#L113) ). Looks like removal would have no impact.